lib/simp/rake/build/unpack.rb

```diff
@@ -50,7 +50,7 @@ def define_tasks
         File.exist?(args.iso_path) or
           fail "Error: You must provide the full path and filename of the ISO image."
 
-        %x{file #{iso_path}}.split(":")[1..-1].to_s =~ /ISO/ or
+        %x{file --keep-going '#{iso_path}'}.split(":")[1..-1].to_s =~ /ISO/ or
```

Comment:
On my dev  CentOS 8.3 server.  I executed the file command to the CentOS-8.3.2011 ISO file and it did not need `--keep-going`.  What is your configuration for which you had this problem?